.changelog/unreleased/features/ibc-integration-test/4153-add-ipv6-test.md

```diff
@@ -0,0 +1,2 @@
+- Disable TLS configuration when using IPv6 for the gRPC endpoint.
```

Comment:
@ljoss17 Missed that when reviewing, but it would nice to explain to end-users why we are doing this.

Reply:
Nevermind, just saw it's an issue for the integration test component.